internal/driver/cli.go

```diff
@@ -66,6 +67,7 @@ func parseFlags(o *plugin.Options) (*source, []string, error) {
 	flagTools := flag.String("tools", os.Getenv("PPROF_TOOLS"), "Path for object tool pathnames")
 
 	flagHTTP := flag.String("http", "", "Present interactive web based UI at the specified http host:port")
```

Comment:
"interactive web based UI" -> "interactive web UI" to make it a bit shorter and consistent with description of the new flag one line below.

Reply:
Done.